### ISSUE TITLE
CONVEX hull kills performance

### ISSUE THREAD

**Issue by casigus** [TheGameCreators/GameGuruRepo#3980]
been reporting this for over a year.

Convex hulls kill my performance, and this is a completely different PC, i did swap the convex to block and it vastly improved it but with every single update it just updates the objects causing the whole thing to start up again. Every time reported this just been overlooked.

![image](https://user-images.githubusercontent.com/68471279/208255669-1c106f95-9b81-4340-8797-26045057ea66.png)

**Comment by davetgc** [comment]
Hi,

Are you saying that the collisions are being reset after manual changes?

**Comment by casigus** [comment]
yeah!. they are. but thats because the update also then updates all the collsions settings which meeans you have to go back and change every single placed one back to something that doesn't kill performance. I dont undeerstand why these objects are set as corvex hulls or what ever the name is when a wall should just be set as a box. It literally freezes my entire ganme build. :(
